### PR TITLE
Update wang2018dynamic_2.md

### PR DESCRIPTION
**Pull request by specfazhou**

### CODE REVIEW

File: _summaries/wang2018dynamic_2.md [hunk 1]
```diff
@@ -1,19 +1,41 @@
...
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+* What is the core idea? <br/>
+In this paper, the authors come up with a new neural network module called EdgeConv that can use local geometric information of point clouds, which is permutational invariant and translational invariant. By dynamically updating a graph of relationships, EdgeConv is able to perform semantic segmentation task. EdgeConv can also easily be integrated into other point cloud processing pipelines. The following picture shows the entire model architecture for classification and segmentation,<br/>
+<img src = 'model_arch.png'>
+
+
+* How is it realized (technically)? <br/>
```
Comment (jaywhang) [inline _summaries/wang2018dynamic_2.md:16]: Before jumping into the expressions below, maybe the functions/variables can be defined first.   Unless you've already read the paper, I don't think the following diagram and equations really make sense.  You could also include Table 1 from the paper for further clarity.

File: _summaries/wang2018dynamic_2.md [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
+EdgeConv has achieved great results in classification, part segmentation, and semantic segmentation. For classification, the authors evaluate their model on the ModelNet40, which not only achieved the state-of-the-art result, but also cost much shorter time to train. For segmentation, their model archieved competitive results to other point cloud processing methods. The following three pictures show how their model performs:
+<img src = 'dyn_result1.png'>
+<img src = 'dyn_result2.png'>
+<img src = 'dyn_result3.png'>
+
+
+
+
+* What interesting variants are explored?<br/>
+One very interesting point is that you can regard PointNet, PointNet++, MoNet, and PCNN as special cases with special choices of edge features and aggregation operations.
```
Comment (jaywhang) [inline _summaries/wang2018dynamic_2.md:36]: Perhaps this belong to the earlier section where you describe different choices of edge feature and aggregation operation.  I don't think they are "variants" of the method itself, since they correspond to other existing methods in the literature.

File: _summaries/wang2018dynamic_2.md [hunk 3]
```diff
@@ -1,19 +1,41 @@
...
+
+
+* What interesting variants are explored?<br/>
+One very interesting point is that you can regard PointNet, PointNet++, MoNet, and PCNN as special cases with special choices of edge features and aggregation operations. 
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* The EdgeConv module gives local information of point clouds
```
Comment (jaywhang) [inline _summaries/wang2018dynamic_2.md:39]: This is also true for PointNet.  The main advantage seems to be that it gives both local and global information.

File: _summaries/wang2018dynamic_2.md [hunk 4]
```diff
@@ -1,19 +1,41 @@
...
+* EdgeConv can be easily used to other different point clouds processing methods
+* using dynamic graph of relationships
```
Comment (jaywhang) [inline _summaries/wang2018dynamic_2.md:41]: This point is not really discussed earlier -- perhaps another small section that can be added?

File: _summaries/wang2018dynamic_2.md (hunk 3 above)
Comment (specfazhou) [reply _summaries/wang2018dynamic_2.md]: I see, you are right, thanks! already added

File: _summaries/wang2018dynamic_2.md [hunk 5]
```diff
@@ -1,19 +1,41 @@
...
+score: 8/10
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+* What is the core idea? <br/>
+In this paper, the authors come up with a new neural network module called EdgeConv that can use local geometric information of point clouds, which is permutational invariant and translational invariant. By dynamically updating a graph of relationships, EdgeConv is able to perform semantic segmentation task. EdgeConv can also easily be integrated into other point cloud processing pipelines. The following picture shows the entire model architecture for classification and segmentation,<br/>
```
[the remark this answers is not in the document]
Comment (specfazhou) [reply _summaries/wang2018dynamic_2.md]: I see, thanks! already changed

File: _summaries/wang2018dynamic_2.md (hunk 1 above)
Comment (specfazhou) [reply _summaries/wang2018dynamic_2.md:16]: Thanks for letting me know this problem. I think I probably just remain the written summary in this way, but I will mention functions/variables of EdgeConv first when I do my presentation.

File: _summaries/wang2018dynamic_2.md (hunk 2 above)
Comment (specfazhou) [reply _summaries/wang2018dynamic_2.md:36]: Yeah, I know. But it seems the paper doesn't mention other related works. But thanks for pointing it out, I will state it explicitly when I do my presentation

**Comment by zhaoyue-zephyrus** [comment]
@specfazhou Could you do me a favor by adding a prefix `wang2018dynamic_2_*` to the figure names? e.g. `wang2018dynamic_2_first_choice.png` or `wang2018dynamic_2a.png`. This will make naming more consistent. (Also don't forget to change the hyperlinks in the markdown). Thank you.

**Comment by specfazhou** [comment]
Sure, I will edit them tonight.

Best,
Zhou
________________________________

**Comment by zhaoyue-zephyrus** [comment]
@specfazhou Good. I'll merge them once you rename all figures.

**Comment by specfazhou** [comment]
> @specfazhou Good. I'll merge them once you rename all figures.

Sorry, I don't think I can find the place to change my file name

**Comment by zhaoyue-zephyrus** [comment]
@specfazhou Can you use `git mv ${oldname}.png ${newname}.png` and then commit?

**Comment by specfazhou** [comment]
> @specfazhou Can you use `git mv ${oldname}.png ${newname}.png` and then commit?

Ok, I have a better idea. I will re-uploaded all my pictures again

**Comment by specfazhou** [comment]
> @specfazhou Can you use `git mv ${oldname}.png ${newname}.png` and then commit?

Okay, I changed the names of these pictures already.

**Comment by zhaoyue-zephyrus** [comment]
> > @specfazhou Can you use `git mv ${oldname}.png ${newname}.png` and then commit?
> 
> Ok, I have a better idea. I will re-uploaded all my pictures again

Then the previous ones are still there ....

**Comment by specfazhou** [comment]
> > > @specfazhou Can you use `git mv ${oldname}.png ${newname}.png` and then commit?
> > 
> > 
> > Ok, I have a better idea. I will re-uploaded all my pictures again
> 
> Then the previous ones are still there. Please use `git rm` and then commit.

I think I delete previous pictures all. I know "git mv" is a better way, but to be honest, I don't even know how to download the files on github to my local machine, so............

**Review by zhaoyue-zephyrus — APPROVED** [review]
